### ISSUE TITLE
Remove `fs-promise` dependency

### ISSUE THREAD

**Issue by lukehedger** [MaiaVictor/swarm-js#14]
Not sure why both `fs-promise` and `fs-extra` are dependencies but `fs-promise` is deprecated. `fs-extra` should be able to replace all instances.

```
npm WARN deprecated fs-promise@2.0.3: Use mz or fs-extra^3.0 with Promise Support
```

**Comment by lukehedger** [comment]
This will be fixed by #13

**Comment by evertonfraga** [comment]
#13 and #24 addresses this issue. Please let me know of any further inconvenience.
